app/models/container_image.rb

```diff
@@ -1,10 +1,19 @@
+require 'MiqContainer/MiqContainer'
+
 class ContainerImage < ActiveRecord::Base
```

Comment:
@matthewd @blomquisg I removed the K words.
Anyway since now we just have docker images I would rather spare the effort of creating an STI (db migration, etc) and allow `docker_id` to live here.